src/utils/satelliteUtils.ts

```diff
@@ -3,16 +3,20 @@ import SatelliteInterface from '../utils/types/satelliteInterface'
 import SatelliteDocument from '../utils/types/satelliteInterface'
 export default class SatelliteUtilities {
 
-    static async createSatelite(newSatelite: SatelliteInterface): Promise<SatelliteInterface> {
+    static async createSatelite(newSatelite: SatelliteInterface): Promise<SatelliteDocument> {
         return Satellite.create(newSatelite)
     }
 
     static getSatelliteById(id: string): Promise<SatelliteDocument | null> {
         return Satellite.findById(id).exec();
     }
 
-    static getSatelliteByName(name: string): Promise<SatelliteDocument | null> {
-        return Satellite.findOne({ officialName: name }).exec()
+    static async getSatelliteByName(name: string): Promise<SatelliteDocument | null> {
+        const foundSatellites = await Satellite.fuzzySearch(name);
```

Comment:
Would be probably better to return a list of satellities.

Reply:
Fix it, I'm approving in advance as the code seems fine.

Reply:
+1 here but I'm not approving until it's done